faq.md

```diff
@@ -15,8 +15,7 @@ No other automotive update system has been designed to work in such rigorous sit
 
 
 ### **How does Uptane work with other systems and protocols?**
-
-Other mechanisms for performing updates, such as XXX, YYY, ZZZ, are compatible with Uptane. Uptane can use any mechanism for data transport and is designed to provide strong security guarantees even if the underlying network or transport mechanism is compromised. If an automaker wants to move to a secure update system, keeping their existing update system as a transport for Uptane is an effective way to do so.
+Other mechanisms for performing updates, such as those offered by Red Bend, Movimento, and Tesla, are compatible with Uptane. Uptane can use any mechanism for data transport and is designed to provide strong security guarantees even if the underlying network or transport mechanism is compromised. If an automaker wants to move to a secure update system, keeping their existing update system as a transport for Uptane is an effective way to do so.
```

Comment:
```suggestion
Other mechanisms for performing updates, such as those offered by Red Bend, Movimento, and Tesla, are compatible with Uptane in the following manner. Uptane can use any mechanism for data transport, and is designed to provide strong security guarantees even if the underlying network or transport mechanism is compromised. If an automaker wants to move to a secure update system, keeping their existing update system as solely a transport mechanism for Uptane is an effective way to do so.
```

Reply:
It's still a bit unclear what we prescribe exactly until one finishes reading the whole paragraph. If there is a way to clarify upfront that existing update mechanisms should be used _solely_ as a transport mechanism for Uptane, which is then responsible for security, then we should do it.

Reply:
Good point, Trishank. I've tweaked your suggested text below.
```suggestion
Other mechanisms for performing updates, such as those offered by Red Bend, Movimento, and Tesla, are compatible with Uptane for handling certain tasks that are not related to security. For example, Uptane can use any mechanism for data transport, and is designed to provide strong security guarantees even if the underlying network or transport mechanism is compromised. If an automaker wants to improve the security of its operations, keeping its existing update system as a transport mechanism for Uptane is an effective way to do so.
```

Reply:
I would tweak it as follows:

> Other mechanisms for performing updates, such as those offered by Red Bend, Movimento, and Tesla, are compatible with Uptane for solely handling data transport. Uptane can use any mechanism for data transport, and is designed to provide strong security guarantees even if the underlying network or transport mechanism is compromised. If an automaker wants to improve the security of its operations, keeping its existing update system as a transport mechanism for Uptane is an effective way to do so.

A bit redundant, which could use some fixing, but you get the idea.

Reply:
How about the following? Its your suggested text with a few stray words removed.
```suggestion
Other mechanisms for performing updates, such as those offered by Red Bend, Movimento, and Tesla, are compatible with Uptane solely for handling data transport. Uptane can use any transport mechanism, and still provide strong security guarantees, even if the underlying network or transport mechanism is compromised. If an automaker wants to move to a secure update system, keeping their existing system as a transport mechanism for Uptane is an effective way to do so.
```